### PR TITLE
[Workers] more errors documentation

### PR DESCRIPTION
**Pull request by deadlypants1973**
PCX-8083

### CODE REVIEW

File: content/workers/observability/log-from-workers.md [hunk 1]
```diff
@@ -140,126 +140,4 @@ Note that:
...
-
-async function handleRequest(request) {
-  // An error here will return the origin response, as if the Worker wasn’t present.
-  // ...
-  return fetch(request);
-}
-```
-{{</tab>}}
-{{</tabs>}}
+Refer to the [Workers Logpush documentation](/workers/observability/logpush/) to learn how to create and configure Logpush jobs.
```
Comment (irvinebroque) [inline content/workers/observability/log-from-workers.md]: I think this page should to link to the new errors page being added in this PR, for discoverability. Thoughts?

File: content/workers/platform/limits.md [hunk 2]
```diff
@@ -150,6 +150,10 @@ Routes in fail open mode will bypass the failing Worker and prevent it from oper
 
 Routes in fail closed mode will display a Cloudflare `1027` error page to visitors, signifying the Worker has been temporarily disabled. Cloudflare recommends this option if your Worker is performing security related tasks.
 
+### Loop limit
+
+A Worker cannot call itself or another Worker more than 16 times. In  order to prevent infinite loops between Workers, the [`CF-EW-Via`](/fundamentals/reference/http-request-headers/#cf-ew-via) header's value is an integer that indicates how many invocations are left. Every time a Worker is invoked, the integer will decrement by 1. If the count reaches zero, a [`1019`](/workers/observability/log-from-workers/#error-pages-generated-by-workers) error is returned.
+
```
Comment (irvinebroque) [inline content/workers/platform/limits.md]: I don't think this should be in limits. It's not a limitation of the platform in that "here is a valid thing that you want to do, but you might hit this ceiling" — it's a safeguard, feels more appropriate as part of errors content (which is great to have in its own section)

File: content/workers/observability/errors.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+## Error pages generated by Workers
+
+When a Worker running in production has an error that prevents it from returning a response, the client will receive an error page with an error code, defined as follows:
+
+{{<table-wrap>}}
+
+| Error code | Meaning                                                                                                           |
+| ---------- | ----------------------------------------------------------------------------------------------------------------- |
+| `1101`     | Worker threw a JavaScript exception.                                                                              |
+| `1102`     | Worker exceeded [CPU time limit](/workers/platform/limits/).                                                      |
```
Comment (RebeccaTamachiro) [inline content/workers/observability/errors.md]:
```suggestion
| `1102`     | Worker exceeded [CPU time limit](/workers/platform/limits/#cpu-time).                                                      |
```

File: content/workers/observability/errors.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+| `1015`     | Worker hit the [burst rate limit](/workers/platform/limits/#burst-rate).                                          |
+| `1019`     | Worker hit [loop limit](/workers/platform/limits/#loop-limit).                                                    |
```
Comment (RebeccaTamachiro) [inline content/workers/observability/errors.md]:
```suggestion
| `1019`     | Worker hit [loop limit](#loop-limit).                                                    |
```

Comment (RebeccaTamachiro) [reply content/workers/observability/errors.md]: Not sure at all about this one, just noticed that the anchor link that is there currently is not working and then saw this same page has an h3 called `Loop limit`

**Review by RebeccaTamachiro — APPROVED** [review]

**Pull request merged**